### PR TITLE
Linter added to JS Package

### PR DESCRIPTION
**Pull request by excerebrose**
1. Linter added
2. Fixed vars

Fixes changes mentioned in https://github.com/hCaptcha/hmt-escrow/pull/230

related to issue https://github.com/hCaptcha/hmt-escrow-form/issues/2

### CODE REVIEW

**Comment by excerebrose** [comment]
@uivlis

File: hmt_escrow_js/Job.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Job {
     this._manifest_url = manifest_url
     this._factory_addr = factory_addr
 
-    this.serialized_manifest = this._download_manifest()
+    this._serialized_manifest = this._download_manifest()
     this.amount = null
```
Comment (uivlis) [inline hmt_escrow_js/Job.js:18]: Interesting, why does `amount` not have a `_` in front of, just like the others?

Comment (excerebrose) [reply hmt_escrow_js/Job.js:18]: stupidity on my part

Comment (uivlis) [reply hmt_escrow_js/Job.js:18]: :) Then can I ask, why do all of them have a `_` in front of them? Are they internal in any way?

Comment (excerebrose) [reply hmt_escrow_js/Job.js]: habit for internals when designing classes, here the usecase is slightly different but I stuck with it

**Pull request merged**